### PR TITLE
SLE Add rsyslog_remote_loghost droping remediations

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add rsyslog_remote_loghost droping remediations

#### Rationale:

- Implement Bash and Ansible remediations that rely on drop-in configuration files

### CODE REVIEW

File: linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_loghost/bash/sle15.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_sle
+
+{{{ bash_instantiate_variables("rsyslog_remote_loghost_address") }}}
+{{{ bash_comment_config_line("/etc/rsyslog.conf", '^*.*') }}}
+{{{ bash_comment_config_line("/etc/rsyslog.d/*.conf", '^*.*') }}}
```
Comment (jan-cerny) [inline linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_loghost/bash/sle15.sh:5]: The dots and asterisks need to be escaped

File: shared/macros/10-bash.jinja [hunk 2]
```diff
@@ -1598,6 +1598,18 @@ fi
 :type format: str
 
 #}}
+
+{{%- macro bash_comment_config_line(config_file, key) -%}}
```
Comment (jan-cerny) [inline shared/macros/10-bash.jinja:1602]: This macro needs to be tested by a BATS test.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: shared/macros/10-bash.jinja (hunk 2 above)
Comment (teacup-on-rockingchair) [reply shared/macros/10-bash.jinja:1602]: Very useful indeed, haven't had the BATS pleasure so far :+1: done in 4438714d16e0939f61d3759949b672b03fa5832f

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**